### PR TITLE
Bump version `v0.2.0`

### PR DESCRIPTION
**Pull request by girianshido**
Version v0.2.0 (09-12-2020)
===========================

- Fix get marketMakers by collateral tokens #10 
- Fix Token info from blockhain and Uniswap-v2 graph #9

### CODE REVIEW

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr14--pregov.review.gnosisdev.com

File: src/App.js [hunk 1]
```diff
@@ -60,7 +25,7 @@ function App() {
                     <Route path='/create/:id' exact>
                         <Details />
                     </Route>
-                    <Route path='/embed/:id/:token1/:token2' exact>
+                    <Route path='/embed/:id/:baseToken/:quoteCurrencyToken' exact>
```
Comment (auryn-macmillan) [inline src/App.js:28]: is there a reason that the quote token is called `quoteCurrencyToken` rather than `quoteToken`?

**Review by auryn-macmillan — APPROVED** [review]
Looks good to me :smile:

File: src/App.js (hunk 1 above)
Comment (girianshido) [reply src/App.js:28]: It was named as `quoteCurrencyToken`. It can be renamed on future releases

**Pull request closed without merging**